code/game/objects/items/devices/scanners_vr.dm

```diff
@@ -0,0 +1,209 @@
+var/global/mob/living/carbon/human/dummy/mannequin/sleevemate_mob
+
+//SleeveMate!
+/obj/item/device/sleevemate
+	name = "\improper SleeveMate 3200"
+	desc = "A hand-held sleeve management tool for performing one-time backups and managing mindstates."
+	icon = 'icons/obj/device_alt.dmi'
+	icon_state = "sleevemate"
+	item_state = "healthanalyzer"
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
+	throwforce = 3
+	w_class = ITEMSIZE_SMALL
+	throw_speed = 5
+	throw_range = 10
+	matter = list(DEFAULT_WALL_MATERIAL = 200)
+	origin_tech = list(TECH_MAGNET = 2, TECH_BIO = 2)
+
+	var/datum/mind/stored_mind
+
+/obj/item/device/sleevemate/attack(mob/living/M, mob/living/user)
+	if(ishuman(M))
+		scan_mob(M, user)
+	else
+		to_chat(user,"<span class='warning'>Not a compatible subject to work with!</span>")
+
+/obj/item/device/sleevemate/attack_self(mob/living/user)
+	if(!stored_mind)
+		to_chat(user,"<span class='warning'>No stored mind in \the [src].</span>")
+		return
+
+	var/choice = alert(user,"What would you like to do?","Stored: [stored_mind.name]","Delete","Backup","Cancel")
+	switch(choice)
```

Comment:
If you add  `if (!stored_mind) return` it will gracefully handle multi-clicking.